api/app/lib/updaters.rb

```diff
@@ -102,7 +102,7 @@ def update_belongs_to(model, name, to_add)
   def update_has_many(model, name, to_add)
     value = relationship_map(model, name, to_add)
     model.send("#{name}=", value)
-    model.send(name.to_s).try(:sort, value)
+    model.send(name.to_s).try(:sort_makers, value)
```

Comment:
I think my confusion here came from getting an argument error on `sort`.  Right now, the only time I see `update_has_many` being hit is when authors/collaborators are added to a project.  Those succeed because they have their own sort methods defined that take an argument, but I think any model that hits `update_has_many` will respond to `.try(:sort)`.

It seems that when the associations in this PR are made, they hit the `update_has_many` method and try to call Ruby's sort with an argument, resulting in the error.

`model.send("makers").try(:sort)
=> [#<Maker id: "347c5c38-cde4-4a1...`
 `model.send("makers").try(:sort, model2)
ArgumentError: wrong number of arguments (given 1, expected 0)
`

Does it make sense to change this symbol to something like, `association_sort`?